Gordon360/Services/RecIM/MatchService.cs

```diff
@@ -241,6 +246,17 @@ public async Task UpdateTeamStats(MatchTeamPatchViewModel vm)
                     .ID;
             }
             await _context.SaveChangesAsync();
+            return teamstats;
+
+        }
+        public async Task<MatchCreatedViewModel> UpdateMatch(int matchID, MatchPatchViewModel vm)
```

Comment:
This is kind of my point with the `...CreatedViewModel` not being the best nomenclature for a basic view model. If you are updating a match and you get a match **created** view model, it might be confusing.